### DISCUSSION TITLE
Github Actions Secrets and Variables Bug

### DISCUSSION THREAD

**Discussion by isaac-lee-mort** [community/community#121819] [Actions]
### Select Topic Area

Bug

### Body

Hi,

I've found a bug with the Environment variables section of Github. When you try to sort by any of the fields, it duplicates the entries and sorts. This appears to happen indefinitely. I've attached a video of this happening.

When refreshing the page, the duplicated variables aren't persistent, so don't appear to have any backend impact, just on the frontend.

MacOS 14.4.1
Chrome 123.0.6312.87

https://github.com/community/community/assets/39830928/4cf4e848-55a8-4f54-a53e-bd8e07e39bbc

**Comment by mininotallarines** [comment]
It's worth noting that I'm only experiencing this in repositories with multiple environments, if you only have one environment, I don't think the bug occurs. I thought it was quite funny though
